static/styles/image_upload_widget.css

```diff
@@ -112,6 +111,20 @@
     }
 }
 
+.user-avatar-section {
```

Comment:
@ArunSankarKs it would to nice if you place this block of code under the user avatar widget CSS section of this file.

Reply:
Okies!